aten/src/ATen/native/cpu/Reduce.h

```diff
@@ -10,6 +10,11 @@ namespace at { namespace native { namespace {
 
 using namespace vec256;
 
+#define VEC_LOOP_HEADER(func_t, data) \
```

Comment:
Note for reviewer: this macro and the functions below were moved unchanged from Loops.h.